src/main/java/org/cbioportal/service/FrontendPropertiesServiceImpl.java

```diff
@@ -234,6 +234,7 @@ private String getPropertyValue(FrontendProperty property) {
             case "query_sets_of_genes":
             case "skin.patient_view.custom_sample_type_colors_json":
             case "oncoprint.clinical_tracks.config_json":
+            case "custom_buttons_json":
```

Comment:
Since the property refers to a JSON file, we need to add it to the readFile() case.